[2023-11-27 07:14:57,975] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_dim_case manual__2023-11-27T07:14:51.994345+00:00 [queued]>
[2023-11-27 07:14:57,987] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: final_project_data_covid_etl.create_dim_case manual__2023-11-27T07:14:51.994345+00:00 [queued]>
[2023-11-27 07:14:57,987] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 07:14:57,987] {taskinstance.py:1357} INFO - Starting attempt 1 of 3
[2023-11-27 07:14:57,987] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-27 07:14:58,004] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_dim_case> on 2023-11-27 07:14:51.994345+00:00
[2023-11-27 07:14:58,009] {standard_task_runner.py:52} INFO - Started process 229 to run task
[2023-11-27 07:14:58,011] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'final_project_data_covid_etl', 'create_dim_case', 'manual__2023-11-27T07:14:51.994345+00:00', '--job-id', '348', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmprvnv89dp', '--error-file', '/tmp/tmpy7ur33hh']
[2023-11-27 07:14:58,012] {standard_task_runner.py:80} INFO - Job 348: Subtask create_dim_case
[2023-11-27 07:14:58,074] {task_command.py:369} INFO - Running <TaskInstance: final_project_data_covid_etl.create_dim_case manual__2023-11-27T07:14:51.994345+00:00 [running]> on host 671693f20ed3
[2023-11-27 07:14:58,172] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ricky
AIRFLOW_CTX_DAG_ID=final_project_data_covid_etl
AIRFLOW_CTX_TASK_ID=create_dim_case
AIRFLOW_CTX_EXECUTION_DATE=2023-11-27T07:14:51.994345+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-27T07:14:51.994345+00:00
[2023-11-27 07:14:58,173] {python.py:173} INFO - Done. Returned value was: None
[2023-11-27 07:14:58,187] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=final_project_data_covid_etl, task_id=create_dim_case, execution_date=20231127T071451, start_date=20231127T071457, end_date=20231127T071458
[2023-11-27 07:14:58,225] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-27 07:14:58,271] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
